src/index.ts

```diff
@@ -219,27 +293,37 @@ export const run = async (config: SimulatorConfig): Promise<void> => {
 
   log.success('starting simulator...');
 
-  if (associate) {
+  if (onboard) {
     config.onConnect = async (deviceId) => {
       log.info(
-        `ATTEMPTING TO ASSOCIATE ${config.deviceId} WITH API KEY ${config.apiKey} VIA ${config.apiHost}`,
+        `ATTEMPTING TO ONBOARD ${config.deviceId} via ${onboard} WITH API KEY ${config.apiKey} VIA ${config.apiHost}`,
       );
 
       // wait to ensure the device is available in AWS IoT so it can be associated
       await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
Should this waiting period change for preconnect @gfleming-nordic ? Is it even necessary?

Reply:
Works right away without. Good catch!

Reply:
I'm unsure but I would hypothesize it's only really needed for JITP association (if at all). It might be worth testing